README.md

```diff
@@ -6,12 +6,12 @@ Skopeo is a command line tool for working with remote image registries.
 The copy command will take care of copying the image from internal.registry to production.registry. If your production registry requires credentials to login in order to push the image, skopeo can handle that as well.
 
 ### Steps to run the Task
-- Create a Kubernetes secrets containining credentials of the registries.
+- Create a Kubernetes secrets containining credentials of the registries. Refer to [official Tekton documention](https://tekton.dev/docs/pipelines/auth/#configuring-docker-authentication-for-docker).
     - <b>Method 1</b>  is to use existing `/home/username/.docker/config.json` (to create this, use `docker login` command). 
       ```
       kubectl create secret generic docker-registry \
-      --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
-      --type=kubernetes.io/dockerconfigjson
+            --from-file=.dockerconfigjson=${HOME}/.docker/config.json \
```

Comment:
Let's leave only method "2" as the preferred example, please.

Reply:
So I should completely remove Method 1?